library/xml-model/src/test/java/org/cru/godtools/xml/model/tract/CardTest.kt

```diff
@@ -68,4 +72,21 @@ class CardTest {
         assertTrue(page.cards[1].isLastVisibleCard)
         assertFalse(page.cards[2].isLastVisibleCard)
     }
+
+    @Test
+    fun testCardBackgroundColorFallbackBehaviorForTractPage() {
+        val page = TractPage(
+            Manifest(),
+            cards = { listOf(Card(it, isHidden = false), Card(it, isHidden = false), Card(it, isHidden = true)) }
+        )
+        val card = Card(page, content = {
+            listOf(
+                InlineTip(it, "tip1"),
+                Paragraph(it, content = { listOf(InlineTip(it, "tip2")) }),
+                InlineTip(it, "tip3"),
+                InlineTip(it, "tip1"),
+            )
+        })
+        assertThat(card.backgroundColor, equalTo(page.cardBackgroundColor))
```

Comment:
I would explicitly set `page.cardBackgroundColor` to something like `Color.GREEN`, that way you know the card color is definitely using the page color.